drivers/serial/uart_mcux_flexcomm.c

```diff
@@ -1090,7 +1091,7 @@ static const struct uart_driver_api mcux_flexcomm_driver_api = {
 };
 
 
-#ifdef CONFIG_UART_INTERRUPT_DRIVEN
+#if defined(CONFIG_UART_INTERRUPT_DRIVEN) || defined(CONFIG_UART_ASYNC_API)
```

Comment:
just one request is to also update the comment on line 1109

Reply:
Done!
It's
`/* defined(CONFIG_UART_INTERRUPT_DRIVEN) || defined(CONFIG_UART_ASYNC_API) */`
now